content/homework.rst

```diff
@@ -22,6 +22,18 @@ Working remotely isn't for everyone, and it can take a while to adjust to it.
 Ask for advice from your team members, they have also been
 through this and can share their experience.
 
+While we don't work from the same location, we are all located
+in a 2 hours difference time zone.
+
+We don't have fixed working our but our customers expect us to be available from **9 to 5 from Monday until Friday**.
```

Comment:
working our -> working hours. Also, perhaps it would help to specify the time zone for the 9 to 5 time frame.